cmd/boulder-observer/README.md

```diff
@@ -204,6 +233,34 @@ successful.
 
 This is configurable, see `buckets` under [root/schema](#schema).
 
+### CRL Metrics
+
+These metrics will be available whenever a valid CRL prober is configured.
+
+#### obs_crl_this_update
+
+Unix timestamp value of the thisUpdate field for a CRL.
```

Comment:
Add some recommendation/ examples here for utilizing this timestamp.

Reply:
done. lmk if the added text seems sufficient